drivers/SmartThings/philips-hue/src/utils/hue_motion_sensor_utils.lua

```diff
@@ -0,0 +1,17 @@
+local utils = require "utils"
+
+local lazy_disco_handlers = utils.lazy_handler_loader("disco")
+---@class SensorUtils
+local motion_sensor_utils = {}
```

Comment:
Looks like this is used for both contact and motion so I think the name (and filename) should be made more general like `hue_sensor_utils`

Reply:
Good catch, I used the IDE refactor facilities but I guess it failed on the filesystem operation.

Reply:
@tpmanley This is done in a PR further down the line now, where we generalize to multi-service devices instead of just generalizing to sensors.